multinet/resolvers/query.py

```diff
@@ -5,11 +5,14 @@
 
 def workspaces(root, info, name=""):
     """Return list of workspace names accessible to the user."""
-    return [
-        workspace
-        for workspace in db.get_workspaces(name)
-        if not name or workspace == name
-    ]
+    if db.get_workspaces(name) is not None:
+        return [
+            workspace
+            for workspace in db.get_workspaces(name)
+            if not name or workspace == name
+        ]
+    else:
+        return
```

Comment:
What was the reason for this change?

Reply:
It was a remnant from earlier attempts. I removed it.